manifests/xbuild.pp

```diff
@@ -0,0 +1,14 @@
+# @summary Install cross-building dependencies
+class poudriere::xbuild {
+  include poudriere
+
+  package { $poudriere::xbuild_package:
+    ensure => installed,
+  }
+
+  service { 'qemu_user_static':
+    ensure => 'running',
+    enable => true,
+    status => '/usr/sbin/binmiscctl lookup arm',
```

Comment:
is this a placeholder for now?

Reply:
Nope, the quem_user_static "service" is only responsible to setup things when it starts, but has no support for status/stop.  We ensure that it's enabled so that this setup is done on reboot, and we also must make sure that it has been started on the host so that qemu is ready to launch these files.

`arm` here is just one of the architectures that is included.